src/main/nsis/bin/process.sh

```diff
@@ -0,0 +1,47 @@
+#/bin/bash
+# change to directory of this script
+cd "$( dirname "${BASH_SOURCE[0]}" )"
```

Comment:
The Windows scripts don't change the working directory. I can see why you're doing it - so the relative path to the .JAR is correct - but that could be better handled with a local variable.

Reply:
Thanks for the review, you are correct, the behaviour must be the same as the Windows scripts. I will modify the PR.